### ISSUE TITLE
Task: First set of batch examples for multiple linalg functions 

### ISSUE THREAD

**Issue by bmwoodruff** [possee-org/genai-numpy#34]
**Description:**

I'll be creating a branch that contains new examples for several functions from the `linalg` module. 

**Acceptance Criteria:**

- [x] For each example below, I'll check the example using the development version of numpy, use the output produced rather than AI's output, and properly format the example.
  - [x] np.linalg.svd
  - [x] np.linalg.outer
  - [x] np.linalg.trace
  - [x] np.linalg.matrix_rank
  - [x] np.linalg.multi_dot
  - [x] np.linalg.vecdot
  - [x] np.linalg.matmul
  - [x] np.linalg.det
  - [x] np.linalg.svdvals - I know @ogidig5 is working on this one. I plan to include it anyway here, though I'll remove it before sending anything off as a PR. 
  - [x] np.linalg.eig
- [x] I'll push the branch to my fork of Numpy. 
- [x] I'll Provide a link to that branch here.

**Comment by bmwoodruff** [comment]
I started page for review at 
- https://github.com/possee-org/genai-numpy/issues/40
I ended up making two branches. The first was my working branch, where I saved things as I went.  The second branch I used rebase and squash so that everything appears in one commit, clean and pretty to send off to Numpy. 
- (messy working branch) https://github.com/numpy/numpy/compare/main...bmwoodruff:numpy:first-batch-examples-linalg
- ([rebased and squashed](https://chatgpt.com/share/b162cbc0-c15b-4a67-aff9-14c9eebbb1e1)) https://github.com/numpy/numpy/compare/main...bmwoodruff:numpy:ai-gen-examples-linalg-batched
